KPI-team1-frontend/src/pages/SettingsPage.tsx

```diff
@@ -113,7 +124,7 @@ export default function UserDetailsPage(): JSX.Element {
                   {circles.map((c) => (
                     <option
                       value={c.circle_user[0].circle_id}
-                    // TODO provide random key
+                      // TODO provide random key
```

Comment:
let's remove the todo now
```suggestion
                      key={`${c.circle_user[0].circle_id}-key`}
```